### PR TITLE
[WIP] Passport SponsorSeller/ParticipatingSeller model

### PR DESCRIPTION
**Pull request by Jeremy-Xue**
WIP model code

Based on RFC: https://docs.google.com/document/d/1HqGgOQoJtvLgstOISq1Ifk6c3N0VCrmmSzqH7LjRZHE

### CODE REVIEW

File: db/migrate/20200811003516_create_participating_sellers.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreateParticipatingSellers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :participating_sellers do |t|
+      t.string :name
+      t.bigint :seller_id, null: true
```
Comment (Jeremy-Xue) [inline db/migrate/20200811003516_create_participating_sellers.rb:5]: actually I just realized, I think null defaults to true, should the other options (name stamp_url etc) be set to null:false instead?

Comment (jacoblb64) [reply db/migrate/20200811003516_create_participating_sellers.rb:5]: sure, makes sense to me, whatever's default can be default, no need to manually specify

File: app/models/sponsor_seller.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+#  logo_url    :string
+#  name        :string
+#  reward      :string
+#  reward_cost :integer
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#  location_id :bigint
+#
+class SponsorSeller < ApplicationRecord
+    validates_presence_of :logo_url, :name, :reward, :reward_cost
```
Comment (justintmckibben) [inline app/models/sponsor_seller.rb:15]: I think :logo_url, :name, :reward you don't need to validate the presence of. Definitely not :logo_url and :reward — but I can see the argument of you wanting to validate name is there

File: db/migrate/20200811003054_create_sponsor_sellers.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+class CreateSponsorSellers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :sponsor_sellers do |t|
+      t.string :name
+      t.bigint :location_id
+      t.string :logo_url
+      t.string :reward
+      t.integer :reward_cost
```
Comment (justintmckibben) [inline db/migrate/20200811003054_create_sponsor_sellers.rb:8]: I'd default this to three, and make it nullable: false

File: app/models/participating_seller.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+#  id             :bigint           not null, primary key
+#  name           :string
+#  stamp_url      :string
+#  tickets_secret :string
+#  created_at     :datetime         not null
+#  updated_at     :datetime         not null
+#  seller_id      :bigint
+#
+class ParticipatingSeller < ApplicationRecord
+    validates_presence_of :name, :tickets_secret
```
Comment (justintmckibben) [inline app/models/participating_seller.rb:14]: tickets_secret you don't need to validate the presence of. Matter of fact we want for it to be nullable

**Pull request merged**